exolve-m.js

```diff
@@ -349,6 +349,8 @@ function Exolve(puzzleSpec,
              <b>Ctrl-B:</b> Print crossword, <b>Ctrl-/:</b> Jump to/back-from
              notes, <b>Ctrl-*:</b> Mark clue as fave in notes, adding a *
              prefix.</li>
+         <li><b>Delete:</b>
+						 Clear the content sof the current square.</li>
```

Comment:
Delete the extraneous leading space. Typo: s/content sof/contents of/

Reply:
updated